cmd/client/client_integration_test.go

```diff
@@ -96,7 +96,10 @@ func mustGetFreePort() string {
 	defer l.Close()
 
 	// Get the port number from the address that the Listener is listening on.
-	addr := l.Addr().(*net.TCPAddr)
+	addr, ok := l.Addr().(*net.TCPAddr)
```

Comment:
Changed cause of forcetypeassert linter
https://github.com/gostaticanalysis/forcetypeassert